functions/transformations/sum.go

```diff
@@ -105,8 +106,8 @@ type SumIntAgg struct {
 	sum int64
 }
 
-func (a *SumIntAgg) DoInt(vs []int64) {
-	for _, v := range vs {
+func (a *SumIntAgg) DoInt(vs *array.Int64) {
+	for _, v := range vs.Int64Values() {
```

Comment:
The arrow arrays have special methods for computing the sum. Can you use those methods here? Those methods are written using SIMD instructions.